examples/website/pointcloud/app.js

```diff
@@ -123,7 +123,7 @@ export default class App extends PureComponent {
   _onLoad({header, loaderData, attributes, progress}) {
     // metadata from LAZ file header
     const {maxs, mins} =
-      loaderData.header.mins && loaderData.header.maxs
+      loaderData.header && loaderData.header.mins && loaderData.header.maxs
```

Comment:
```suggestion
    loaderData?.header.mins && loaderData?.header.maxs
```
Maybe we can use `mesh.header.boundingBox?`

Reply:
The issue here occurs on the draco dataset. It doesn't contain "header"

Reply:
I see. The `loaderData` is loader specific and should not be used for generic processing. I have a PR up that fixes the mesh typing.

Reply:
But for a fix on release branch this is OK.

Reply:
Done